.travis.yml

```diff
@@ -13,32 +12,20 @@ cache:
 
 matrix:
   include:
-    # Main tests on supported Python versions.
-    - python: "2.7"
-      env: TOXENV=py27
-    - python: "3.4"
-      env: TOXENV=py34
-    - python: "3.5"
-      env: TOXENV=py35
-    - python: "3.6"
-      env: TOXENV=py36
-    - python: "pypy-5.3.1"
-      env: TOXENV=pypy
-
-    # Linting, docs, and other non-test stuff.
-    - python: "3.4"
-      env: TOXENV=lint
-    - python: "3.5"
-      env: TOXENV=docs
-    - python: "2.7"
-      env: TOXENV=packaging
-
-    # Test we haven't broken our major dependencies.
-    - python: "2.7"
-      env: TOXENV=py27-twistedMaster
-
+    - python: 2.7
+      dist: trusty
+    - python: 3.4
+      dist: trusty
+    - python: 3.5
+      dist: trusty
+    - python: 3.6
+      dist: trusty
+    - python: 3.7
+      dist: xenial
+    - python: pypy-5.3.1
+      dist: trusty
```

Comment:
Are we not running the docs tox env anymore?

Reply:
tox-travis does it for you (it runs all the tox-envs it can given the environment) so see the 3.7 CI job.

Reply:
I don't see the docs env being run anywhere... https://travis-ci.org/python-hyper/hyper-h2/jobs/481283520

Maybe that was the reason why I avoided tox-travis, because it doesn't fully run the environments if you don't specify them explicitly.

Reply:
Ah, you are right - I had it configured incorrectly (and never noticed this over the last couple of years on another project). I've changed it now, however I think the matrix and tox is better. Shall I get rid of tox-travis?

Reply:
Updated to use tox as before, `tox-travis` is worse - I'll fix Hyperframe.